portal-impl/src/portal.properties

```diff
@@ -609,6 +609,16 @@
     #
     portlet.preferences.strict.store=true
 
+##
+## Portlet Captcha
+##
+
+    #
+    # Env: LIFERAY_MAX_PERIOD_CHALLENGES_PERIOD_BLACKLISTED_PERIOD_PORTLET_UPPERCASEN_AMES
```

Comment:
based on the discussion in https://liferay.atlassian.net/browse/LPD-1558?focusedCommentId=2569258,
Stian thinks: "I recognize the example configuration + upgrade scenario that you highlighted. So I would not use configuration for the blacklist. It would be more effective to hardcode this list."
Therefore, I am going to use portal.properties to hardcode it so far

Reply:
For this decision, I think it would be up to App Sec and the Security Team to decide.

Technically, the same scenario may still come into play, where:
- A portal admin sets their own portal property in portal-ext.properties
- We eventually have a new security fix and add a new portlet to the list
- However, since the portal admin already set their own portal property, the new portlet would not be included.

And so, I would suggest confirming whether we want a hard-coded list in a java class or whether we want to proceed with making this configurable by the portal admin.